NGitLab/IPipelineScheduleClient.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using NGitLab.Models;
+
+namespace NGitLab;
+
+public interface IPipelineScheduleClient
+{
+    /// <summary>
+    /// All the scheduled pipelines of the project
+    /// </summary>
+    IEnumerable<PipelineScheduleBasic> All { get; }
+
+    /// <summary>
+    /// Return the details of single schedule
```

Comment:
This is a property, so better not start the description with a verb (mostly reserved for methods).

```suggestion
    /// Details for a single schedule
```